vanities/vanities.go

```diff
@@ -0,0 +1,12 @@
+package vanities
+
+// Vanity Contains the vanity web url.
+type Vanity struct {
+	WebURL string `json:"webUrl"`
+}
+
+// GetVanity Given (data tbc) it will lookup the vanity url and return it as a Vanity struct.
```

Comment:
The vanity function is currently a no-op; i'll make the url configurable once it's implemented properly